portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -110,6 +139,16 @@ public void deleteComment(long id) {
         return modifyWithQuery(query);
     }
 
+    /**
+     * Sorts comments from datastore by the rating in descending order and return first 'n'.
+     * @param n        Number of comments to return.
+     * @return List of pairs (Comment id, Comment object), sorted by the rating in descending order.
+     */
+    public ArrayList<Map.Entry<Long, Comment>> sortByRating(int n) {
```

Comment:
Use a more explicit name for your argument, like numberOfComments which you used in the other methods